vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/treegrid/demo/TreeGridDemo.java

```diff
@@ -146,4 +149,78 @@ public boolean hasChildren(Account item) {
 
         addCard("TreeGrid with lazy loading", grid);
     }
+
+    // Drag and drop
+    private Department draggedItem;
+
+    private void createDragAndDrop() {
+        DepartmentData departmentData = new DepartmentData();
+        // begin-source-example
+        // source-example-heading: Drag and drop
```

Comment:
There's no comments on any of this code. At minimum document what part of the code does what.

Reply:
Done.